src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -1,5 +1,8 @@
-{
+﻿{
   "dependencies": {
+    "Microsoft.Build.Framework": "0.1.0-preview-00022",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
```

Comment:
The only reason I have it here, is because I wanted to access the "UpdateGitHubFileAsync" method from VersionsRepoUpdater.cs.  I can look at moving this to the Tasks project though.